.github/workflows/codeql.yml

```diff
@@ -0,0 +1,149 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  schedule:
+    - cron: '44 9 * * 5' # Runs every Friday at 9:44 AM
```

Comment:
is there a reason for this timing?

Reply:
Came from the template. I thought it doesn't matter when (but how often), so I left it as is